client/src/app/site/pages/organization/pages/dashboard/dashboard.subscription.ts

```diff
@@ -15,7 +15,9 @@ export const meetingFields: (keyof Meeting)[] = [
     `template_for_organization_id`,
     `description`,
     `location`,
-    `organization_tag_ids`
+    `organization_tag_ids`,
+    `locked_from_inside`,
+    `user_ids`
```

Comment:
Again. Check if this is really necessary.

Reply:
Reverted, s.o.